src/panel/direction/DirectionForm.jsx

```diff
@@ -56,7 +56,17 @@ export default class DirectionForm extends React.Component {
       destinationInputText,
     } = this.props;
 
+    const isMobile = isMobileDevice();
+
+    const VehicleSelectorComponent = props => <VehicleSelector
```

Comment:
I don't see the point of declaring a function here. Why not using `VehicleSelector` directly?